src/components/panel/panel.js

```diff
@@ -1160,3 +1222,129 @@ MdPanelPosition.prototype.right = function(opt_right) {
 MdPanelPosition.prototype.getRight = function() {
   return this._right;
 };
+
+
+
+/*****************************************************************************
+ *                               MdPanelAnimation                            *
+ *****************************************************************************/
+
+
+/**
+ * Animation configuration object. To use, create an MdPanelAnimation with the
```

Comment:
Let's add all this documentation to the public documentation section above. I think it would be useful for users to know how to use the component. (Less useful for our internal docs.)